docs/source/agent-framework/core-service-agents/master-driver/master-driver-agent.rst

```diff
@@ -46,314 +39,32 @@ Required libraries:
 The easiest way to install the requirements for drivers included in the VOLTTRON repository is to use ``bootstrap.py``
 (see :ref:`platform installation for more detail <Platform-Installation>`)
 
+.. code-block:: bash
+
+   python bootstrap.py --drivers
+
 
 Master Driver Agent Configuration
 ---------------------------------
 
 The Master Driver Agent configuration consists of general settings for all devices. The default values of the Master
-Driver should be sufficient for most users.  The user may optionally change the interval between device scrapes with the
-driver_scrape_interval.
-
-The following example sets the driver_scrape_interval to 0.05 seconds or 20 devices per second:
+Driver should be sufficient for most users.
```

Comment:
get rid of should be ...

just add as follows or some other wording that says look at the example.